### ISSUE TITLE
patch_match compiling failed

### ISSUE THREAD

**Issue by ZeroCool22** [lkwq007/stablediffusion-infinity#48]
I already execute both commands:

pip install opencv-python

conda install -c conda-forge opencv

Still giving that error.

**Comment by ZeroCool22** [comment]
![Screenshot_19](https://user-images.githubusercontent.com/13344308/193942182-6a857953-1422-4cbe-83bb-2fe7bed875fb.jpg)

Mmmm, strange, looks like it's working even with that error?

**Comment by lkwq007** [comment]
@ZeroCool22 
If you got `patch_match compiling failed` error, the `patchmatch` option then falls back to `edge_pad`. 

Please try the latest version (via a fresh clone). I think it might work on Windows 10/11 x64.

**Comment by ZeroCool22** [comment]
> @ZeroCool22 If you got `patch_match compiling failed` error, the `patchmatch` option then falls back to `edge_pad`.
> 
> Please try the latest version (via a fresh clone). I think it might work on Windows 10/11 x64.

You mean i clone the repo again? Or you refer to some command in the CMD?

**Comment by lkwq007** [comment]
> > @ZeroCool22 If you got `patch_match compiling failed` error, the `patchmatch` option then falls back to `edge_pad`.
> > Please try the latest version (via a fresh clone). I think it might work on Windows 10/11 x64.
> 
> You mean i clone the repo again? Or you refer to some command in the CMD?

Right. You may need to clone the repo again. (delete the original one and clone it again / or clone the repo at a new location) 
Due to a submodule change, I am unsure if the submodules can be updated correctly via commands .
